ci-operator/step-registry/mirror-images/qe-test-images/OWNERS

```diff
@@ -0,0 +1 @@
+../OWNERS
```

Comment:
Since you are the owner of this step, I suggest to put your name in the approver section. (you can remove this link and create `OWNERS` file instead).

Reply:
@xiuwang , please add me as an approver in your step `OWNERS` file, thanks!